### ISSUE TITLE
Error: Column `variableField` doesn't exist when run Cohort Diagnostics demo script

### ISSUE THREAD

**Issue by gongliangz** [OHDSI/CohortDiagnostics#1106]
Hi team,
When I ran Cohort Diagnostics demo script (from Drs. Rao and Gilbert, 2023-11-30), an error message occurred in the last step (see below). Can anyone help me out? Thanks 

launchDiagnosticsExplorer(sqliteDbPath = "MyCohortDiagnosticsResulst.sqlite")
Loading required package: shiny
Using DatabaseConnector jdbc driver.
Initalizing pooled connection
Connecting using SQLite driver

Listening on http://127.0.0.1:5065
Warning: Error in tidyr::pivot_wider: Can't subset columns that don't exist.
✖ Column `variableField` doesn't exist.
  83: <Anonymous>
  82: signalCondition
  81: signal_abort
  80: cnd_signal
  79: handlers[[1L]]
  78: <Anonymous>
  77: signalCondition
  76: signal_abort
  75: abort
  74: stop_subscript
  73: stop_subscript_oob
  72: <Anonymous>
  71: vctrs::vec_as_location
  70: chr_as_locations
  69: as_indices_impl
  68: as_indices_sel_impl
  67: vars_select_eval
  63: eval_select_impl
  62: tidyselect::eval_select
  61: build_wider_spec
  60: pivot_wider.data.frame
  59: tidyr::pivot_wider
  58: dplyr::mutate
  57: dplyr::mutate
  56: dplyr::mutate
  55: dplyr::group_by
  54: dplyr::arrange
  53: dplyr::mutate
  52: dplyr::filter
  51: dplyr::select
  50: %>%
  49: getDatabaseMetadata
  48: databaseInformationModule
  47: module
  42: callModule
  41: shiny::moduleServer
  40: OhdsiShinyModules::cohortDiagnosticsServer
  39: server [C:\Users\gxz108\AppData\Local\R\win-library\4.3\CohortDiagnostics\shiny\DiagnosticsExplorer/server.R#2]
   2: shiny::runApp
   1: launchDiagnosticsExplorer
Error in tidyr::pivot_wider(., id_cols = c("startTime", "databaseId"),  : 
  Can't subset columns that don't exist.
✖ Column `variableField` doesn't exist.

**Comment by laurenrichter** [comment]
Not sure what the fix is, but wanted to chime in that I'm getting the same error.

**Comment by gongliangz** [comment]
No update yet. sorry

**Comment by hussef01** [comment]
Any solution on this? Also running into the same error :(

**Comment by lrasmus** [comment]
Also ran into the same error - appreciate knowing I'm not the only one seeing this.  Will report back if I'm able to find any information looking through the code.

**Comment by lrasmus** [comment]
Okay - a few updates from my end.  As usually happens, this was a "me" problem, but the errors made me think there was an issue in the code (with apologies to jumping to that conclusion - I should know by now OHDSI packages are solid).

I was getting the original error because my SQLite database was empty.  Running `createMergedResultsFile` "succeeded" (in that I didn't get an error) but when I looked at it all that was happening was the migration scripts were running to create the database structure.

Why wasn't the database being filled?  I didn't read the instructions.  I was pointing `createMergedResultsFile` to a directory with CSVs in it, but I missed that it was looking for a ZIP file in the directory I pointed to.

Why wasn't there a ZIP file?  That I'm going to chalk up to not managing my executions of the package better.  I know I got it to run.  But I was also trying different parameters at different times.  So at some point I must have run it or didn't let it fully run, etc. and thought it completed but the ZIP file was never created.  I took a step back, did a clean run and the ZIP file was there.

I realize this is a lot of detail when the problem was me the whole time.  Hopefully if anyone comes across this issue in the future, my embarrassing mistakes can help them out.

**Comment by laurenrichter** [comment]
> Okay - a few updates from my end. As usually happens, this was a "me" problem, but the errors made me think there was an issue in the code (with apologies to jumping to that conclusion - I should know by now OHDSI packages are solid).
> 
> I was getting the original error because my SQLite database was empty. Running `createMergedResultsFile` "succeeded" (in that I didn't get an error) but when I looked at it all that was happening was the migration scripts were running to create the database structure.
> 
> Why wasn't the database being filled? I didn't read the instructions. I was pointing `createMergedResultsFile` to a directory with CSVs in it, but I missed that it was looking for a ZIP file in the directory I pointed to.
> 
> Why wasn't there a ZIP file? That I'm going to chalk up to not managing my executions of the package better. I know I got it to run. But I was also trying different parameters at different times. So at some point I must have run it or didn't let it fully run, etc. and thought it completed but the ZIP file was never created. I took a step back, did a clean run and the ZIP file was there.
> 
> I realize this is a lot of detail when the problem was me the whole time. Hopefully if anyone comes across this issue in the future, my embarrassing mistakes can help them out.

Good catch, same issue for me. I ran the packages as instructed from the start, but executeDiagnostics() did not create a .zip file and instead only generated the .csvs (I'm using MacOSX/RStudio). I created a .zip manually and was able to run the Shiny app without a problem afterwards.  Would recommend that the vignette be updated to indicate that you might have to create your own .zip file depending on the system (even if you're only looking at once database).
